### ISSUE TITLE
Crash After Multiple Searches Using Large # Of Accounts

### ISSUE THREAD

**Issue by andibuch** [BrianEstrada/PokeScanner#215]
I'm having a crash occur for new that I wanted to bring to your attention before you push out an official 1.8.0

This seems to occur on both the alpha that you posted to the sub, as well as an unofficial me recent build of the alpha that was posted earlier today

The crash occurs when I've configured a lot of accounts. I've been playing with 19 and 37 accounts, as these are the optimal account for a 3 and 4 radius respectively. It has occurred in each, but seems to be occurring more frequently with the 37 accounts.

For the crash, I have been working with 3 or 4 radius searches. The crash seems to happen sooner on the 4 radius searches.

To make it happen, I search an area, wait for it to finish, then search a new area.

On 37 account 4 radius searches, after the third search it will reliably crash. It sometimes takes more searches with less accounts and smaller radii.

I will say the crash happens regardless of whether I wait between searches. Even if I search, wait 10 seconds, switch apps and switch back, the crash will still happen.

Thoughts?

**Comment by BrianEstrada** [comment]
yeah the crash is related to the database it isn't meant to handle that many connections.

**Comment by andibuch** [comment]
Is there a simple way to modify the database to account for large numbers
of connections before the release?

If not, what would you say is the maximum number of connections recommended
at the moment to avoid these crashes?

On Aug 3, 2016 3:55 PM, "Brian Estrada" <notifications@github.com> wrote:

yeah the crash is related to the database it isn't meant to handle that
many connections.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/BrianEstrada/PokeScanner/issues/215#issuecomment-237352429>,
or mute the thread

**Comment by BrianEstrada** [comment]
somewhere between 20 and 30...and no from what it looks like there is not an easy way :/

**Comment by dusktildawn48** [comment]
hey brian, im not having this same issure, but didnt feel like it needed its own issure. under account manager all my accounts are just loading. they are all ptc

**Comment by BrianEstrada** [comment]
^ Click on the refresh button to the top right

**Comment by andibuch** [comment]
Hmmm... the crash seems to happen to me on 19 accounts too, though, just less frequently

**Comment by andibuch** [comment]
Is there a way for me to get crash logs to you?  Would that be helpful?  I'm new to developing / debugging on android, but am tech savy with a history doing QA at game companies, so I'd be happy to help

**Comment by rhari991** [comment]
This project has crashlytics installed, so whenever the app crashes on any phone, the crash report is sent to us. So no worries :)

**Comment by jomarocas** [comment]
The project no working maybe recently niantics force me update the app and in other github issue related with new update and maybe the api changes in niantic, for update the api
